assets/app/scripts/filters/resources.js

```diff
@@ -28,7 +56,20 @@ angular.module('openshiftConsole')
         }
       }
       if (resource && resource.metadata && resource.metadata.annotations) {
-        return resource.metadata.annotations[key];
+        var mappings = annotationMap[key];
```

Comment:
I would go even further and check if the requested annotation is in `resource.metadata.annotations`... if it is, return it. If it's not, check the `annotationMap`